Syntaxes/Haskell-SublimeHaskell.sublime-syntax

```diff
@@ -304,7 +304,7 @@ contexts:
         - include: expression_stuff
 
   function_signature:
-    - match: '^(\s*)(?!--|\b(let|in|case|where)\b)(?:(\(\W+\)|[\w'']+)|[\(\[])(?=[\w'',\s\[\]\(\)]*(?:\s*)((?:::)|\u2237))'
+    - match: '^(\s*)(?!--|\b(let|in|case|where)\b)(?:(\(\W+\)|[\w'']+))(?=[\w'',\s\[\]\(\)]*(?:\s*)((?:::)|\u2237))'
```

Comment:
I'm not entirely sure what the point of this was. It'd be great to get an example of something using this branch, otherwise, I can't think of a reason to keep this.

If it is necessary, I can revert, but there should probably be a case to exit `function_signature` when finding a closing paren/bracket